src/components/Global/ImageGrid.vue

```diff
@@ -37,7 +37,7 @@ watch(() => props.images, () => {
   props.images.forEach(image => {
     if (image.basename && !(image.basename in imageDetails.value)) {
       imageDetails.value[image.basename] = ref('')
-      const url = image.thumbnail_url || ''
+      const url = image.thumbnail_url || image.smallThumbUrl || ''
```

Comment:
the first thumbnail_url is from images generated in the server

Reply:
so it tries to first get images from the server and if that doesn't work where are .smallThumbUrl images coming from? the thumbnail service?

Reply:
so smallThumbUrl is what is set in the settings.js `loadAndCacheImages` function now, if it is set in the archive response. Its a direct link to download the small thumbnail from S3. If it doesn't exist we just set empty string, which triggers us to use the thumbnail service as a fallback